.github/workflows/bundle-size.yml

```diff
@@ -0,0 +1,23 @@
+name: Compressed Size
+
+on:
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+    build:
+        name: Bundle size check
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use desired version of NodeJS
+              uses: actions/setup-node@v2
+              with:
+                  node-version-file: '.nvmrc'
+                  cache: npm
+            - uses: preactjs/compressed-size-action@b03ec15e1a93cb5c062d66a1c263f9ddbd2f81ef
```

Comment:
Can we add a step name here, so it won't print out `Run preactjs/compressed-size-action@b03ec15e1a93cb5c062d66a1c263f9ddbd2f81ef` on the job logs.

Something like "Checking for file changes"

Reply:
Great suggestion! I have added a step name.